doc/source/changes/version_0_26.rst.inc

```diff
@@ -321,3 +328,13 @@ Fixes
   the array. Now the result will be nan for that array but the operation will carry on for other arrays.
 
 * fixed stacking arrays with anonymous axes.
+
+* fixed background color in the viewer when using filters in the `compare()` dialog (closes :editor_issue:`66`)
+
+* fixed autoresize of columns by double clicking between column headers (closes :editor_issue:`43`)
+
+* fixed representing a 0D array (scalar) in the viewer (closes :editor_issue:`71`)
+
+* fixed viewer not displaying an error message when saving or loading a file failed (closes :editor_issue:`75`)
+
+* fixed viewer showing array with empty axes names and labels when starting a new session (closes :editor_issue:`84`)
```

Comment:
* isn't this the same issue than the 0D array issue above (71)?
* did those two issues actually exist in 0.25.2? I don't remember. The issues we introduced in 0.26 (before fixing them), should not be mentioned in the changelog.

Reply:
issue 84 and 71 are a bit different. 
Issue 71 concerns array of size 1 but no axes.
Issue 84 concerns "empty" array, i.e. of size 0 but with potentially fake axes ( try arr = LArray([]) )